tests/integration_tests/modules/test_jinja_templating.py

```diff
@@ -19,12 +20,15 @@ def test_runcmd_with_variable_substitution(client: IntegrationInstance):
     """Test jinja substitution.
 
     Ensure we can also substitute variables from instance-data-sensitive
-    LP: #1931392
+    LP: #1931392 and that underscore-delimited aliases exist for hyphenated
```

Comment:
Nit: I think we should probably still keep the LP at the bottom of the docstring.